biosys/apps/main/api/serializers.py

```diff
@@ -76,6 +76,8 @@ def update(self, instance, validated_data):
         if has_data:
             different_type = instance.type != validated_data.get('type')
             different_data_package = instance.data_package != validated_data.get('data_package')
+            #TODO: implement a smart risk-checking of changing dataset schema when there's data.
+            different_data_package = False
```

Comment:
I've disabled the guard against changing dataset schema if there is data. For now.